docs/usage/getting-started/running.md

```diff
@@ -2,8 +2,8 @@
 
 As end user, you can choose from these ways to run Renovate:
 
-- You use the Mend Renovate App, or
-- You self-host Renovate, for example by running the pre-built Docker image, or
+- You use the Mend Renovate App
+- You self-host Renovate - e.g. Renovate CLI, Renovate Docker image, or Renovate Community Edition / Enterprise Edition
```

Comment:
I think the list of methods to self-host Renovate should be moved somewhere else in the document.

The list item is hard to read because there's too much information in a single line.

Reply:
I might come back and refactor this page. For now, I would love to have the out-of-date OnPrem info updated to reflect CE/EE.